gulpfile.js

```diff
@@ -2,6 +2,7 @@ var gulp = require('gulp'),
     babel = require('gulp-babel'),
```

Comment:
The tasks in here are getting pretty hard to follow, lots going on - can you please start adding a docblock on top of the new tasks describing what they do?